plugins/modules/azure_rm_networkinterface.py

```diff
@@ -374,8 +395,13 @@
                     sample: default
                 load_balancer_backend_address_pools:
                     description:
-                        - List of existing load-balancer backend address pools to associate with the network interface.
+                        - List of existing load-balancer backend address pools associated with the network interface.
+                    type: list
+                application_gateway_backend_address_pools:
+                    description:
+                        - List of existing application gateway backend address pools associated with the network interface.
                     type: list
+                    version_added: "1.10.0"
```

Comment:
It would have been better if sample had been added!
```suggestion
                    sample: ***
```